app/build.gradle

```diff
@@ -43,8 +43,11 @@ dependencies {
     androidTestImplementation 'com.android.support.test:runner:1.0.2'
     androidTestImplementation 'com.android.support.test.espresso:espresso-core:3.0.2'
 
```

Comment:
import libraries : 
first for snackbar
second for floating button
the other one is just an update